### DISCUSSION TITLE
Stop downgrading the feed to a twitter fake timeline

### DISCUSSION THREAD

**Discussion by bionade24** [community/community#13411] [Other Feature Feedback, Questions, & Ideas]
It's probably the most hated thing about Twitter, excluding the NFT BS, that you don't actually have a normal Timeline anymore. Stop trying to get as worse as others, or sooner or later everyone will go off your platform. Maybe just start working on you backend's reliability instead of the frontend design.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
